templates/membership/join.html.twig

```diff
@@ -135,7 +135,7 @@
                             {{ form_widget(form.conditions) }}
                                 <label class="form form__label" for="{{ form.conditions.vars.id }}">
                                     <span class="text__checkbox">
-                                            Oui, j'adhère à la <a href="https://storage.googleapis.com/en-marche-prod/documents/adherents/1-charte-et-manifeste/regles_de_fonctionnement_LaREM.pdf" target="_blank">charte des valeurs, aux statuts et aux règles de fonctionnement</a> de La République En Marche, ainsi qu'aux <a href="{{ path('page_mentions_legales') }}" target="_blank">conditions générales d'utilisation</a> du site
+                                            Oui, j'adhère à la <a href="https://storage.googleapis.com/en-marche-prod/documents/adherents/1-charte-et-manifeste/regles_de_fonctionnement_LaREM.pdf" target="_blank">charte des valeurs, aux statuts et aux règles de fonctionnement</a> de La République En Marche, ainsi qu'aux <a href="{{ path('page_mentions_legales') }}" target="_blank">conditions générales d'utilisation</a> du site et à la <a href="https://en-marche.fr/politique-protection-donnees" target="_blank">politique de protection des données à caractère personnel</a> du site
```

Comment:
is it static page `https://en-marche.fr/politique-protection-donnees`? can you use `path('route_name')` to generate the URL?